client/src/ui/components/list/EntityList.tsx

```
@@ -57,18 +58,25 @@ export const EntityList = ({
-                <ArrowRight className="w-2 fill-current" />
+                <div className="flex flex-col space-y-2">
+                  <div className="flex flex-row justify-between items-center">
+                    <div className="flex flex-row space-x-1 items-center">
+                      {entity?.position && <ViewOnMapIcon className={"my-auto"} position={entity.position} />}
+                      <h5>{entity.name}</h5>
+                    </div>
+                    <ArrowRight className="w-2 fill-current" />
+                  </div>
+
+                  <div className="border border-gold/20 bg-gold/10 rounded p-2">
+                    {entityContent && entityContent(entity.id)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)